.gitignore

```diff
@@ -0,0 +1,8 @@
+*.pyc
+*.pyo
+*~
+*.swp
+build
+.DS_Store
+lib/modules/
+lib/pypy.vm.*
```

Comment:
please remove these "lib" reference here, IIUC there's left over from the old repo.